packages/components/molecules/f-searchbox/package.json

```diff
@@ -59,10 +59,10 @@
     "vuex": "3.5.1"
   },
   "devDependencies": {
-    "@justeat/f-button": "2.0.0-beta.0",
+    "@justeat/f-button": "2.0.0",
     "@justeat/f-error-message": "0.3.1",
     "@justeat/f-globalisation": "0.2.0",
-    "@justeat/f-mega-modal": "1.0.0-beta.0",
+    "@justeat/f-mega-modal": "1.0.0",
```

Comment:
```suggestion
    "@justeat/f-mega-modal": "1.1.0",
```

Reply:
@DamianMullins Interesting! Would that work in the same PR if that version is not yet published?

Reply:
@zlatinivanov-je because it is monorepo, it works. To publish the components, f-mega-modal should be published first